contrib/cirrus/runner.sh

```diff
@@ -133,6 +133,13 @@ function _run_endpoint() {
     make endpoint
 }
 
+function _run_minikube() {
+    # Assume test details live under 'test/minikube', if not, remove the next line.
+    _bail_if_test_can_be_skipped test/minikube
+    msg "This is just a stub for now."
```

Comment:
```suggestion
bats test/minikube/001-kube.bats
```

I think?

Reply:
@cevich can I actually remove this completely? 

I want to add the new minikube tests to https://github.com/containers/podman/blob/main/test/system/710-kube.bats so I can use the helper functions from helpers.bash.

Reply:
I have no problem with you changing whatever is needed.  Topic-wise, this is definitely a system-test, I'm just not sure about all the ramifications or requirements of including it in **those** system tests.

For example as I understand it, those tests are run in the Fedora/RHEL testing systems.  So, a very different environment entirely from Cirrus-CI, and our VMs.  So that's definitely something you'd want to talk with @edsantiago about.

A hack-ish idea could be to have it in a separate directory but symlink the helper script.  Not sure how Ed would feel about that, since it could become hard to remember the helpers have multiple usage-contexts.  Then again, any breakage should show up by a CI test failing, so maybe it's not so bad.